packages/blocks/src/api/raw-handling/index.js

```diff
@@ -16,9 +16,16 @@ import figureContentReducer from './figure-content-reducer';
 import shortcodeConverter from './shortcode-converter';
 import { deepFilterHTML, getBlockContentSchema } from './utils';
 
-import { getPhrasingContentSchema } from './phrasing-content';
-
 export { pasteHandler } from './paste-handler';
+
+/**
+ * WordPress dependencies
+ */
+import { getPhrasingContentSchema } from '@wordpress/dom';
+
+/**
+ * @deprecated since 5.6. Callers should import getPhrasingContentSchema directly from '@wordpress/dom' package.
+ */
 export { getPhrasingContentSchema };
```

Comment:
Was this a public API of "blocks". If it's the case, we can proxy it and add  a `deprecated` call?

Reply:
I believe that's what I did, I will add updated README files to this PR

Reply:
Ah, you meant the function call, not merely the annotation - sure!